gui/src/components/Backups.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+    <div>
+        <h4 class="py-3">Backups in Cluster: {{cluster.cluster_name}}</h4>
+        <ClusterSummary :cluster="cluster"/>
+        <div class="row justify-content-end mb-3 pr-3 py-3">
+            <button class="btn btn-primary mx-1" data-toggle="modal" data-target="#registerBackup">Create Backup
+            </button>
+            <button
+                    type="button"
+                    class="btn btn-primary mx-1"
+                    @click="viewRestoreHistory()"
+            >View Restores
+            </button>
+        </div>
+        <table class="table text-center table-bordered">
+            <thead class="table-secondary">
```

Comment:
If there are no backups, the backup table is empty. 
It's personal taste but I find it a bit settling to see an empty table. Without visual cues it makes me wonder if the content was properly loaded or not.
Maybe a text displayed below the header that says something like
"There are no backup yet" or "No backup" would be welcome.
![Screen Shot 2020-07-01 at 18 07 14](https://user-images.githubusercontent.com/3835021/86226111-252f5880-bbc6-11ea-8286-77c7bf7ab5f4.png)

Reply:
That's a good idea.